package.json

```diff
@@ -38,27 +38,27 @@
     "prepare": "husky install"
   },
   "peerDependencies": {
-    "bootstrap": "^5.2.1",
-    "popper.js": "^1.16.1"
+    "bootstrap": "5.2.1",
+    "popper.js": "1.16.1"
   },
   "devDependencies": {
-    "@11ty/eleventy": "^2.0.0",
-    "@commitlint/cli": "^17.1.2",
-    "@commitlint/config-conventional": "^17.1.0",
-    "autoprefixer": "^10.4.12",
-    "husky": "^8.0.1",
-    "lint-staged": "^13.0.3",
-    "nodemon": "^2.0.20",
-    "npm-run-all": "^4.1.5",
-    "postcss": "^8.4.16",
-    "postcss-cli": "^10.0.0",
-    "prettier": "^2.7.1",
-    "purgecss": "^5.0.0",
-    "sass": "^1.55.0",
-    "serve": "^14.1.1",
-    "standard-version": "^9.5.0",
-    "stylelint": "^15.2.0",
-    "stylelint-config-twbs-bootstrap": "^7.0.0"
+    "@11ty/eleventy": "2.0.0",
+    "@commitlint/cli": "17.1.2",
+    "@commitlint/config-conventional": "17.1.0",
+    "autoprefixer": "10.4.12",
+    "husky": "8.0.1",
+    "lint-staged": "13.0.3",
+    "nodemon": "2.0.20",
+    "npm-run-all": "4.1.5",
+    "postcss": "8.4.16",
+    "postcss-cli": "10.0.0",
+    "prettier": "2.7.1",
+    "purgecss": "5.0.0",
+    "sass": "1.55.0",
+    "serve": "14.1.1",
+    "standard-version": "9.5.0",
+    "stylelint": "15.2.0",
+    "stylelint-config-twbs-bootstrap": "7.0.0"
```

Comment:
Fixing these in place solved issues when running npm i and is what we do in repos like otis.